code/modules/surgery/bodyparts/_bodyparts.dm

```diff
@@ -55,27 +55,35 @@
 	/// For limbs that don't really exist, eg chainsaws
 	var/is_pseudopart = FALSE
 
-	///If disabled, limb is as good as missing.
+	// Limb disabling variables
+	///Controls if the limb is disabled. TRUE means it is disabled (similar to being removed, but still present for the sake of targeted interactions).
 	var/bodypart_disabled = FALSE
-	///Multiplied by max_damage it returns the threshold which defines a limb being disabled or not. From 0 to 1. 0 means no disable thru damage
-	var/disable_threshold = 0
-	///Controls whether bodypart_disabled makes sense or not for this limb.
-	var/can_be_disabled = FALSE
-	///Multiplier of the limb's damage that gets applied to the mob
+	///The percentage of damage taken against max limb damage required to disable a limb. Any value between 0 and 1.1 enables limb disabling via damage.
+	var/disabling_threshold_percentage = 0
```

Comment:
Why is this an integer here but you use it as a boolean on the prosthethic limbs? You use it with calculations so please keep it an integer. 

Could you also make the comment more clear? 0 means no disable, higher than that is the percentage of max limb damage untill it gets disabled (where 1 = 100%). Why does this comment imply that you can't go higher than 1.1, I don't see why not?

Reply:
Oh I just forgot the robotic limbs when I reverted it from a boolean, because I initially misunderstood what this var meant. That's my bad.

The reason any value above 1 wouldn't do anything is because limbs literally cannot exceed their maximum damage value. It clamps the damage. If the value was above 100%, it would be impossible to achieve.

Reply:
Ah okay, then don't say 1.1, just say 1 is max and 0 means no disable and anything inbetween is percentage of limb damage needed to disable

Reply:
I'll clarify it a bit better no problem 👍